ansible/group_vars/all

```diff
@@ -20,6 +20,8 @@ airflow_firstname: "Kielipankki"
 airflow_lastname: "User"
 airflow_email: "ling-admin@listat.csc.fi"
 
+restic_repository: "s3:https://a3s.fi/nlf-harvester-backup"
```

Comment:
Arguably this isn't a backup, but an entire versioning repository. Should the name reflect that?

Reply:
It should, I'll change that!